apps/public-docsite-v9/src/Icons/Icons.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import { SendRegular } from '@fluentui/react-icons';
+import descriptionMd from './IconsDescription.md';
+
+export { Default } from './IconsDefault.stories';
+export { BundleIcon as bundleIcon } from './IconsBundleIcon.stories';
+export { Styling } from './IconsStyling.stories';
+
+export default {
+  title: 'Icons/Overview',
+  component: SendRegular,
+  parameters: {
+    docs: {
+      description: {
+        component: descriptionMd,
+      },
+    },
+    videos: [
+      {
+        href: 'https://www.youtube.com/watch?v=XSPwgmUZ4Tw',
+        preview: '/social/fluent-trainings-ep03.webp',
```

Comment:
the image doesn't work on PR deploed site [Icons / Overview - Default ⋅ Storybook (windows.net)](https://fluentuipr.z22.web.core.windows.net/pull/30783/public-docsite-v9/storybook/index.html?path=/docs/icons-overview--default) 

<img width="235" alt="Screenshot 2024-03-19 at 15 09 42" src="https://github.com/microsoft/fluentui/assets/28751745/825e701f-7f7d-4a59-9c23-f6722eb10143">
It worked after I changed the img src to './social' instead of '/social'

Reply:
Makes sense, good catch!